### ISSUE TITLE
API Performance Issues

### ISSUE THREAD

**Issue by StefanKandlbinder** [opendata-stuttgart/meta#104]
Hallo,
the last couple of days i am experiencing performance issues (very slow, failure) when doing requests like https://api.luftdaten.info/v1/filter/type=SDS011&area=48.323368,14.298756,50.
Any ideas?
Anybody else?

**Comment by papics** [comment]
Yes, I am seeing the same behaviour, the first really bad period was between ~0300UT an ~1400UT on the 5th of January, then today since ~0830UT. Single sensor requests are as slow as filter type requests. Seems ike a general issue. :(

**Comment by papics** [comment]
This problem is pretty recurring, the API right now is not returning any data (since ~8:39 UT). There were short 0.5-1 hour down times in the past days, but this is the longest for a few weeks.
